### ISSUE TITLE
Seems to be very few noVNC params that can be supported

### ISSUE THREAD

**Issue by whoyz** [geek1011/easy-novnc#11]
When I try to add the parameter "--novnc-params quality=8" or "--novnc-params compression=8", I get those error: 

Error: error parsing noVNC params: unknown option "quality"
Error: error parsing noVNC params: unknown option "compression".

Seems to be very few noVNC params that can be supported?

**Comment by pgaskin** [comment]
Yes, for now. When I get around to working on the next version, I'll be completely revamping the configuration for more NoVNC and authentication options.
